src/issuance.rs

```diff
@@ -470,7 +470,7 @@ impl IssueBundle<Unauthorized> {
 
 impl<T: IssueAuth> IssueBundle<T> {
     /// Returns the reference notes for the `IssueBundle`.
-    pub fn get_reference_notes(self) -> HashMap<AssetBase, Note> {
+    pub fn get_reference_notes(&self) -> HashMap<AssetBase, Note> {
```

Comment:
I noticed that the `get_reference_notes` function was introduced in PR #124 within a new `impl<T: IssueAuth> IssueBundle<T> { ... }` block. Since there's already the same `impl` block above ([issuance.rs#L205](https://github.com/QED-it/orchard/blob/update_asset_supply/src/issuance.rs#L205)), maybe we could move `get_reference_notes` to the existing block to consolidate related implementations.

Reply:
yes, let's consolidate.